lib/index.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
     AWS.config.update(awsConfig);
 
     const S3 = new AWS.S3({
+      endpoint: config.endpoint,
```

Comment:
According to documentation endpoint should be an instance of AWS.Endpoint:
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#endpoint-property
Does it work fine with plain string?

I assume AWS uses the default endpoint if config.endpoint is null and there are no issues with that?

Reply:
Actually it is:
```
(property) ServiceConfigurationOptions.endpoint?: string | AWS.Endpoint
The endpoint URI to send requests to. The default endpoint is built from the configured region. The endpoint should be a string like 'https://{service}.{region}.amazonaws.com' or an Endpoint object.
```

And i'm using `http://sber***.s***1.***cloud.ru`

Reply:
https://github.com/aws/aws-sdk-js/blob/master/lib/service.d.ts#L74

Reply:
Ok, I see.